### DISCUSSION TITLE
Is there a way to change UI Size

### DISCUSSION THREAD

**Discussion by Ge0ps** [flxzt/rnote#1267] [Q&A]
The UI design, i.e. the buttons and such, are way to small for my taste. Is there a way to change UI size?

**Comment by Doublonmousse** [comment]
Are you on windows ? If yes, what's your display scaling ?
If it's something between 125 % and 175 %, that's a known upstream issue of gtk4.
Basically gtk does not support fractional scaling on windows (yet, see https://gitlab.gnome.org/GNOME/gtk/-/issues/1036), so it uses 100% scaling even on a 175% display scaling setting, making the interface very small.

There's some way to force the UI to be larger (force the UI to a 2 scale, hence 200 % scaling) by setting `GDK_SCALE = 2` in the code though in that case it will be a little too large on the other direction (200 % on a 175% or 150% display scaling).
It's more of a workaround though (not everything plays nicely) so it's not in the windows version for now. That _may_ come in a future version courtesy of gtk4 updating the windows backend to fix this issue
